<launch>
    <node pkg="ba_code" type="object_tracker.py" name="object_tracker" output="screen"></node>
    <node pkg="ba_code" type="object_scout.py" name="object_scout" output="screen"></node>
    <node pkg="ba_code" type="object_visualizer.py" name="object_visualizer" output="screen"></node>
    <node pkg="tf" type="static_transform_publisher" name="camera_map_transform_fixed" args="0 0 0.102 0 0 0 base_link map 10" />
</launch>